### ISSUE TITLE
Publish the package on `crates.io`

### ISSUE THREAD

**Issue by datdenkikniet** [Doctave/doctave#26]
I noticed that this package is not published on `crates.io`. I think it'd be good to add this, to simplify installation and ensure that there will be a nice backup.

Is this something that could be done?

**Comment by begleynk** [comment]
Yeah definitely something could be done! Maybe we could release the next version to crates.io too? Looking at the docs it looks like it's fairly straightforward to setup.

**Comment by begleynk** [comment]
Going to take this out of the v0.4 milestone since this will require a bit more work. The doctave-markdown package that this project depends upon also needs to go into Crates.io. Nothing against it, it just requires a bit more coordination on my part.

Should get this setup in the near future though.

In the meantime, you can use Cargo to manage doctave with the following:

```
cargo install --git https://github.com/Doctave/doctave --tag 0.4.0
```
